CMakeLists.txt

```diff
@@ -29,4 +29,49 @@ export_var(LEATHERMAN_LIBRARIES)
 if(LEATHERMAN_ENABLE_TESTING)
     enable_testing()
     add_subdirectory(tests)
+
+    add_custom_target(cppcheck COMMAND cppcheck --enable=warning,performance --error-exitcode=2 --quiet ${CPPCHECK_DIRS})
+
+    include(FindPythonInterp)
```

Comment:
I'm fine with ticketing it for later.

On Thu, Feb 26, 2015 at 2:59 PM, Branan Purvine-Riley <
notifications@github.com> wrote:

> In CMakeLists.txt
> <https://github.com/puppetlabs/leatherman/pull/3#discussion_r25474014>:
>
> > @@ -29,4 +29,49 @@ export_var(LEATHERMAN_LIBRARIES)
> >  if(LEATHERMAN_ENABLE_TESTING)
> >      enable_testing()
> >      add_subdirectory(tests)
> > +
> > +    add_custom_target(cppcheck COMMAND cppcheck --enable=warning,performance --error-exitcode=2 --quiet ${CPPCHECK_DIRS})
> > +
> > +    include(FindPythonInterp)
>
> Yeah, it's probably worth factoring out just like the CFLAGS stuff is done
> in #2 <https://github.com/puppetlabs/leatherman/pull/2>. The number of
> things that are changing in the cfacter repo for the "move to leatherman"
> commit is getting pretty long, though. I'm starting to feel like it's worth
> doing a cleanup/de-dup pass as a next step, rather than trying to get it
> all reorganized in the first pass.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/puppetlabs/leatherman/pull/3/files#r25474014>.
>